### DISCUSSION TITLE
Memory saver

### DISCUSSION THREAD

**Discussion by anandspathak** [jgromes/RadioLib#440] [General]
Sir,
Is it possible to compile library just for lora sx1278 so to avoid other radios compile to save memory???
Thanks,
Regards
Anand Pathak
India

**Comment by jgromes** [comment]
Only the modules that are used in your code will be compiled, the rest will be optimized away, so there's no extra memory being used.

**Reply by anandspathak** [reply]
Thank you for reply,
Regards
Anand Pathak
